src/zenhub.py

```diff
@@ -19,17 +19,17 @@ def main():
     repo_name = sys.argv[1]
     issue = sys.argv[2]
 
-    zen = ZenHub(repo_name=repo_name,
-                 issue=issue)
+    zen = ZenHub(repo_name=repo_name, issue=issue)
     print(json.dumps(zen.get_info()))
 
 
 class ZenHub():
```

Comment:
We should all discuss the structure of classes. Does a ZenHub instance represent an issue, a board, etc?